Source/Core/Core/IOS/USB/Bluetooth/BTReal.cpp

```diff
@@ -376,8 +393,12 @@ void BluetoothRealDevice::SendHCIResetCommand()
   u8 packet[3] = {};
   const u16 payload[] = {HCI_CMD_RESET};
   memcpy(packet, payload, sizeof(payload));
-  libusb_control_transfer(m_handle, REQUEST_TYPE, 0, 0, 0, packet, sizeof(packet), TIMEOUT);
-  INFO_LOG_FMT(IOS_WIIMOTE, "Sent a reset command to adapter");
+  const int ret =
+      libusb_control_transfer(m_handle, REQUEST_TYPE, 0, 0, 0, packet, sizeof(packet), TIMEOUT);
+  if (ret < 0)
```

Comment:
`if (ret < LIBUSB_SUCCESS)` ?

Reply:
Same with the others

Reply:
Note that here, `ret` indicates the number of bytes transferred if it's nonnegative, but it's probably best to change it to `LIBUSB_SUCCESS` anyways (the `<` versus `!=` still applies for that difference).

I can also change a few `< 0` comparisons to `< LIBUSB_SUCCESS` in GCAdapter.cpp.

Reply:
Ah ok.  I probably could go either way on that too.  This works though.